crates/parcel_plugin_rpc/src/nodejs/rpc_host_nodejs.rs

```diff
@@ -90,4 +106,31 @@ impl RpcHost for RpcHostNodejs {
 
     Ok(Arc::new(RpcConnectionNodejsMulti::new(connections)))
   }
+
+  fn fs_read_to_string(&self, path: &Path) -> anyhow::Result<String> {
```

Comment:
Can these be internal to the file system, so that some of the logic is a bit more self-contained? This would require `call_rpc` being public on the trait, rather than these specific messages. I wonder if a bit more could be done so that the rpc host doesn't become monolithic too 🤔

Reply:
I totally agree, I've been thinking about this all day and I ended up coming up with [this solution](https://github.com/parcel-bundler/parcel/pull/9802).

Rather than using an enum to describe all messages (which will end up monolithic) - I just have a `RpcHost.send()` method that consumers can use.

The idea is the `FileSystemRpc`, `CacheRpc`, etc, can take the `rpc_host` and call the `send` method directly